### PR TITLE
Add contribution panel title component

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: shared/panels/issue-sandbox-panel/issue-sandbox-panel.tsx [hunk 1]
```diff
@@ -14,7 +15,17 @@ export function IssueSandboxPanel() {
...
   return (
     <Panel>
-      <SidePanelHeader canGoBack={false} canClose={true} title={{ children: "Issues sandbox" }} />
+      <SidePanelHeader
+        canGoBack={false}
+        canClose={true}
+        title={{
+          children: (
+            <ContributionSidepanelTitle badge={{ type: "ISSUE", number: 6789, githubStatus: "OPEN" }}>
+              Issue detail
```
Comment (haydencleary) [inline shared/panels/issue-sandbox-panel/issue-sandbox-panel.tsx:24]: Missing translation

File: shared/features/contributions/contribution-sidepanel-title/contribution-sidepanel-title.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { Typo } from "@/design-system/atoms/typo";
+import { ContributionBadge } from "@/design-system/molecules/contribution-badge";
+
+import { ContributionSidepanelTitleProps } from "./contribution-sidepanel-title.types";
+
+export function ContributionSidepanelTitle({ children, badge }: ContributionSidepanelTitleProps) {
+  return (
+    <div className={"flex w-full flex-row items-center justify-start gap-lg overflow-hidden"}>
+      {!!badge && <ContributionBadge {...badge} />}
```
Comment (haydencleary) [inline shared/features/contributions/contribution-sidepanel-title/contribution-sidepanel-title.tsx:9]: Can't use the side panel header `startContent` for this ?

File: shared/panels/issue-sandbox-panel/issue-sandbox-panel.tsx (hunk 1 above)
Comment (alexbeno) [reply shared/panels/issue-sandbox-panel/issue-sandbox-panel.tsx:24]: Sandbox here

File: shared/features/contributions/contribution-sidepanel-title/contribution-sidepanel-title.tsx (hunk 2 above)
Comment (alexbeno) [reply shared/features/contributions/contribution-sidepanel-title/contribution-sidepanel-title.tsx:9]: will try

Comment (alexbeno) [reply shared/features/contributions/contribution-sidepanel-title/contribution-sidepanel-title.tsx:9]: Nope the start content will centre the title as it's done with the back button

**Review by haydencleary — APPROVED** [review]

**Pull request merged**